bento_beacon/config_files/config.py

```diff
@@ -12,10 +12,12 @@ class Config:
     # version of ga4gh beacon spec, not version of this implementation
     BEACON_API_VERSION = "v2.0.0"
 
+    SMALL_CELL_COUNT_THRESHOLD = int(os.environ.get("BEACON_SMALL_CELL_COUNT_THRESHOLD", 0))
```

Comment:
just a quick thing, but it might be good to have a "safe" default here in case an env var accidentally isn't passed in

Reply:
yes, good point